get_favs.py

```diff
@@ -123,5 +138,6 @@ def main(args):
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description='Get favourites from Twitter')
     parser.add_argument('-m',  action='store_true',help='download all media in each post (photos and video)')
+    parser.add_argument('-f',  action='store_true',help='download media in userid folders')
```

Comment:
the help message suggests `-f` will do the downloading, but it requires `-m`, so maybe something like this makes sense:
```
parser.add_argument('-f', action='store_true',help='store downloaded media in userid folders')
```